public/omni/omni-cluster-setup/registering-machines/join-machines-to-omni.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Join machines to Omni
+---
+
+Omni lets you connect existing Talos nodes, whether they’re running on bare metal, in QEMU, or on cloud instances, to your Omni environment so it can automatically take over their management.
+
+You can do this in two ways:
+
+* **Machine Join Config**: Apply a configuration file to a Talos node to register it with Omni.
+
+* **Kernel Parameters**: Add kernel parameters to your machine’s boot configuration so it automatically connects to Omni during startup.
+
+> **Note:** Once a machine joins Omni, it is managed entirely through Omni.
+> The local Talos API is disabled for security reasons, and all future configuration changes must be made via Omni.
+
+The following sections explain each method in more detail and how to use them.
+
+## Machine Join Config
+
+The Machine Join Config is a lightweight configuration file that lets you  register existing Talos nodes with Omni.
```

Comment:
I guess I won't call it `lightweight configuration`. This is actually a partial config that should always stay with the machine. 

```suggestion
The Machine Join Config is a partial configuration that lets you register existing Talos nodes with Omni.
```